.gitignore

```diff
@@ -64,3 +64,6 @@ deploy-key
 
 # Python Virtual Environments
 venv/
+
+# MacOS
+.DS_Store
```

Comment:
To improve the chances of a faster merge, i would recommend to you, to do this in a separate PR, as i am not 100% sure if this is a wanted change at all.